src/config.json

```diff
@@ -6,27 +6,37 @@
       "name": "Apple TV",
       "id": "appletv",
       "color": "#bcbcbc"
-    }, {
+    },
```

Comment:
Looks like this got hit by my formatter. I can revert these formatting changes or we can leave it as-is :)

Reply:
Please do revert the formatting changes. This file can cause some conflicts when upgrading Jellyfin, so it would be best to keep the change set smaller.